### ISSUE TITLE
"Story" tab on roleplays shows "[NPC] Bartender" as the character in the most recent post

### ISSUE THREAD

**Issue by colearkenach** [RolePlayGateway/roleplaygateway#39]
Regardless of the roleplay, this character appears as the one present in the most recent post despite not being in the most recent post. This appears under all roleplays in the "Story" tab.

**Comment by lostamongtrees** [comment]
Thanks @Jhar226 , going to close this issue as I cannot replicate the error and the story tab has gone bye-bye
